packages/radare2/radare2.0.0.3/opam

```diff
@@ -0,0 +1,60 @@
+opam-version: "2.0"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: [ "Edgar Aroutiounian <edgar.factorial@gmail.com>" ]
+license: "BSD-3-Clause"
+homepage: "https://github.com/fxfactorial/ocaml-radare2"
+dev-repo: "git+https://github.com/fxfactorial/ocaml-radare2.git"
+bug-reports: "https://github.com/fxfactorial/ocaml-radare2/issues"
+build: [
+	["dune" "subst"] {pinned}
+	["dune" "build" "-p" name "-j" jobs]
+]
+depexts: [
+  ["radare2"] {os-family = "debian"}
+  ["radare2"] {os-distribution = "fedora"}
+  ["radare2"] {os-distribution = "arch"}
+  ["radare2"] {os-distribution = "gentoo"}
+  ["radare2"] {os-distribution = "homebrew" & os = "macos"}
+]
```

Comment:
If you have the time I think the depexts could be better in their own `conf-radare2` package. It's fine as is for now but it might be easier for future releases

Reply:
Ok, will do that next time, created issue not to forget: https://github.com/fxfactorial/ocaml-radare2/issues/7